values.yaml

```
@@ -374,7 +378,8 @@ authSidecar:
   port: 8084
 
 loggingSidecar:
-  enabled: false
+  enabled: true
```

The default here should be false, right?

```suggestion
  enabled: false
```

---

Yup, left it for testing purposes. Have reverted it